doc/amazon_sagemaker_model_building_pipeline.rst

```diff
@@ -0,0 +1,872 @@
+
+.. code-block:: python
+
+    step_train = TrainingStep(...)
+    model = Model(
+        image_uri=pytorch_estimator.training_image_uri(),
+        model_data=step_train.properties.ModelArtifacts.S3ModelArtifacts,
+        sagemaker_session=pipeline_session,
+        role=role,
+    )
+
+    # we might also want to create a SageMaker Model
+    step_model_create = ModelStep(
+       name="MyModelCreationStep",
+       step_args=model.create(instance_type="ml.m5.xlarge"),
+    )
+
+    # in addition, we might also want to register a model to SageMaker Model Registry
+    register_model_step_args = model.register(
+        content_types=["*"],
+        response_types=["application/json"],
+        inference_instances=["ml.m5.xlarge"],
+        transform_instances=["ml.m5.xlarge"],
+        description="MyModelPackage",
+    )
+
+    step_model_registration = ModelStep(
+       name="MyModelRegistration",
+       step_args=register_model_step_args,
+    )
+    ...
+
+=================
+Notebooks
+=================
+
+We have a collection of notebooks to demonstrate various use cases with SageMaker model building pipeline. You can find them here:
+
+- `Orchestrate Jobs to Train and Evaluate Models with Amazon SageMaker Pipelines`_
+- `Glue ETL as part of a SageMaker pipeline using Pipeline Callback Step`_
+- `SageMaker Pipelines Lambda Step`_
+- `SageMaker Pipelines integration with Model Monitor and Clarify`_
+- `SageMaker Pipelines Tuning Step`_
+
+.. _Orchestrate Jobs to Train and Evaluate Models with Amazon SageMaker Pipelines: https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-pipelines/tabular/abalone_build_train_deploy/sagemaker-pipelines-preprocess-train-evaluate-batch-transform.ipynb
+.. _Glue ETL as part of a SageMaker pipeline using Pipeline Callback Step: https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-pipelines/tabular/custom_callback_pipelines_step/sagemaker-pipelines-callback-step.ipynb
+.. _SageMaker Pipelines Lambda Step: https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb
+.. _SageMaker Pipelines integration with Model Monitor and Clarify: https://github.com/aws/amazon-sagemaker-examples/tree/main/sagemaker-pipelines/tabular/model-monitor-clarify-pipelines
+.. _SageMaker Pipelines Tuning Step: https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-pipelines/tabular/tuning-step/sagemaker-pipelines-tuning-step.ipynb
+
+============
+Limitations
+============
+
+The SageMaker Model Building Pipeline python SDK offers abstractions to help you construct a pipeline definition at ease. However, there are certain limitations. The followings are the known limitations, if you found out more, please feel free to open a git issue.
```

Comment:
`git`->`github issue` (with added link to repo issue creation)

Reply:
good point.